pkg/kv/kvserver/batcheval/cmd_add_sstable.go

```diff
@@ -38,10 +38,11 @@ func init() {
 	RegisterReadWriteCommand(roachpb.AddSSTable, DefaultDeclareIsolatedKeys, EvalAddSSTable)
 }
 
-var addSSTableRewriteConcurrency = settings.RegisterIntSetting(
+// AddSSTableRewriteConcurrency sets the concurrency of a single SST rewrite.
```

Comment:
nit: I can see that concurrency term came from the pebble code, but isn't that parallelism here? We don't contend for resources and we rewrite multiple blocks in parallel. It trickles down all the way to seams with pebble, but that could be changes later as we please while settings are more user facing. I'll leave it to you if that's worth the change.

Reply:
I think for our purposes the terms are mostly interchangeable. Whether it's truly parallel will depend on e.g. the Go scheduler, OS scheduler, hardware, and workload (setting it to 8 on a single-core machine will not make it parallel). We use both terms in our existing settings, e.g.:

* `kv.bulk_io_write.restore_node_concurrency`
* `kv.rangefeed.max_scan_parallelism`

It's sort of unrelated to this PR in any case, so I'll leave it as-is.

Reply:
I think parallel vs concurrent is more intent rather than execution. With parallel you want you computation to be split and processed independently to get benefit from multiple cores, otherwise there's no point. While concurrent is broader where we want all components run at the same time e.g. multitasking will give lower p90 latency v.s. executing sequentially on parallel requests.